testhelpers/docker_registry.go

```diff
@@ -193,11 +192,37 @@ func (r *DockerRegistry) EncodedLabeledAuth() string {
 	return base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf(`{"username":"%s","password":"%s"}`, r.username, r.password)))
 }
 
-// Add stores the given key name with the provided ImagePrivileges. For example
-// Add("my-image", NewImagePrivileges(Readable, Writable)) will save "my-image" as a
+// SetPrivilegesToImage sets the given image name with the provided ImagePrivileges if ImagePrivileges was enabled.
```

Comment:
What do you mean by "if ImagePrivileges was enabled"?

Reply:
Well, because the method is only exposed in the package scope I added `if WithImagePrivileges was called during registry creation`. Is it clear?